build-system-tests/templates/components/react/cra/index-react-18.tsx

```diff
@@ -0,0 +1,13 @@
+// @ts-nocheck
```

Comment:
Why is `@ts-nocheck` necessary?

Reply:
otherwise there would be a lot of red marks in VS Code.

That might be a good point. Would that prevent some necessary build failures if there's any TS issue?

Reply:
since we're interested in catching TS errors, let's remove it.